### PR TITLE
Deprecate atomic and point users at Podman

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

## Related Bugzillas
-
-

## Related Issue Numbers
-
-

## Pull Request Checklist:

If your Pull request contains new features or functions, tests are required. If the PR is a bug fix and no tests exist, please consider adding some to prevent regressions.
- [ ] Unittests
- [ ] Integration Tests

### CODE REVIEW

**Comment by rhatdan** [comment]
@jlebon @giuseppe @mheon @baude @tomsweeneyredhat PTAL

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+The Atomic tool has been deprecated.
+
+The [Podman](https://podman.io) has replaced it with
```
Comment (jlebon) [inline README.md:3]:
```suggestion
The [Podman](https://podman.io) project has replaced it with
```
?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+The Atomic tool has been deprecated.
```
Comment (jlebon) [inline README.md:1]:
```suggestion
# ⚠️ The Atomic tool has been deprecated. ⚠️
```

**Review by jlebon — COMMENTED** [review]
Minor nits, LGTM otherwise!

File: README.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply README.md:1]: Neat change, I didn't know you could do the warning signs like that.

Comment (TomSweeneyRedHat) [reply README.md:1]: Do we have a date or a RHEL release tie in that the deprecation happened?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
+The [Podman](https://podman.io) has replaced it with
+most of its functionality implemented. Please open issues and questions
+with the [Podman repository](https://github.com/containers/podman).
+
+
```
Comment (TomSweeneyRedHat) [inline README.md:7]: Has this deprecation been noted in the RHEL release notes somewhere?

Comment (rhatdan) [reply README.md:7]: I have no idea.  I am sure it is mentioned that this package is no longer in RHEL8.

**Pull request merged**